src/index.html

```diff
@@ -5,8 +5,514 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <!-- <h1>Catalog</h1> -->
+    <header class="container">
+      <div class="logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="MOYO_logo_img">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Фото
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div data-qa="card" class="card">
+        <div class="card__product_image">
+          <img
+            class="card__product_image_img"
+            src="./images/imac.jpeg"
+            alt="Моноблок APPLE A1419 iMac 27"
+          >
+        </div>
+        <div class="card__product_name">
+          <span>
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </span>
+        </div>
+        <div class="card__product_code">
+          <span>
+            Код товара: 195434
+          </span>
+        </div>
+        <div class="card__product_review">
+          <div class="card__review_stars">
```

Comment:
```suggestion
          <div class="card__review_stars stars--4">
```
Тут еще должен быть модифицированный класс, который указывает сколько активных звезд в блоке

Reply:
++

Reply:
bemlint матюкається: error Tag doesn't have modofiable class [stars--4]